modal/cli/import_refs.py

```diff
@@ -42,6 +44,14 @@ class NoSuchObject(modal.exception.NotFoundError):
     pass
 
 
+class CliUserExecutionError(Exception):
```

Comment:
Just bike-shedding, but this name is not super clear to me.

Reply:
Yeah it's an internal error and won't be ever seen by users. It's not meant to be caught.

Reply:
Yeah I just mean as a developer trying to understand why we have it. Not a big deal though.